client/src/lib.rs

```diff
@@ -220,6 +236,10 @@ pub struct Program<C> {
     sub_client: Arc<RwLock<Option<PubsubClient>>>,
     #[cfg(not(feature = "async"))]
     rt: tokio::runtime::Runtime,
+    #[cfg(feature = "mock")]
+    rpc_client: RpcClient,
```

Comment:
Is `rpc_client` being used anywhere? I think we can make the API a bit nicer depending on whether the user has `async` feature enabled or not.